src/router.ts

```diff
@@ -281,7 +301,24 @@ export class StaticRoute extends Route {
   }
 
   get urlPath() {
-    // TODO: Replace with serving path defined in amagaki.yaml.
-    return `/static${this.podPath}`;
+    return this.servingPath;
+  }
+}
+
+function cleanBasePath(path: string): string {
+  path = path.trim();
+
+  if (!path.startsWith('/')) {
+    path = `/${path}`;
+  }
+
+  while (path.endsWith('/')) {
+    path = path.slice(0, -1);
   }
+
+  return path;
+}
+
+function joinPaths(basePath: string, subPath: string, separator = '/'): string {
+  return [basePath, subPath].join(separator);
```

Comment:
Instead of allocating an array here, can you use string concatenation directly?

Reply:
or alternatively, use path.join() from the standard library

Reply:
I didn't do path.join since I wasn't sure what that would do on windows with the different slashes. But I can simplify it.